src/katgpucbf/fgpu/compute.py

```diff
@@ -95,7 +95,7 @@ def __init__(
             self.pfb_fir = pfb.PFBFIRTemplate(
                 context, taps, self.internal_channels, 32, self.unzip_factor, complex_input=True
             )
-            self.ddc = ddc.DDCTemplate(context, narrowband.taps, narrowband.decimation)
```

Comment:
Reposting as per your request @bmerry, The commit message read 
> Remove redundant field from NarrowbandConfig
It contained both `taps` and `weights`, but `taps` is just
`len(weights)`.

My question was
* Has this always been the case and I've only noticed it because of this explicit commit?
* And/Or is this related to removing the requirement of subsampling % taps == 0?